src/app/story/storyrow/storyrow.component.css

```diff
@@ -17,7 +17,13 @@
     padding-right: 1.5rem;
     position: relative;
     /* Vertically center the text */
-    top: 5px;
+    top: -8px;
+    font-weight: bold;
```

Comment:
Could you apply this style to the first inner div? That way you don't have to use `font-weight: normal !important;` on the `node-timestamp` class.